interfaces/builtin/home_test.go

```diff
@@ -123,6 +124,14 @@ func (s *HomeInterfaceSuite) TestUnexpectedSecuritySystems(c *C) {
 	c.Assert(snippet, IsNil)
 }
 
-func (s *HomeInterfaceSuite) TestAutoConnect(c *C) {
+func (s *HomeInterfaceSuite) TestAutoConnectOnClassic(c *C) {
```

Comment:
Test is failing because the interface is already built by the time the flag is mocked.

Doing this here should fix it:
```
iface := builtin.NewHomeInterface()
restore := release.Mock...etc
defer restore()
c.Check(iface.Auto...etc)